### DISCUSSION TITLE
WayBig agent shows Series and Actor's name with (') as Cast

### DISCUSSION THREAD

**Discussion by dimmitri9** [CodyBerenson/PGMA-Modernized#244] [Q&A] [answered]
WayBig this time, it is recognizing series name as part of the cast, and actor's name with (')

https://www.waybig.com/blog/2015/11/04/luke-adams-fucks-josh-peters-in-my-moms-new-husband-part-6-at-men-com/
![Screenshot 2023-05-12 121529](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/9beb1612-ba15-4503-abe7-98fb73c4076b)
![Screenshot 2023-05-12 121556](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/9d20ba05-3e19-4f23-865a-e5b1cc819e42)
![Screenshot 2023-05-12 121604](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/c90d5c31-b15c-49c7-9939-eb115e220ba1)
![Screenshot 2023-05-12 193440](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/60edce15-f731-489d-9711-4d74adfc0325)

https://www.waybig.com/blog/2015/10/19/rafael-alencar-fucks-dylan-knight-in-my-neighbors-son-part-3-at-men-com/
![Screenshot 2023-05-20 100609](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/a3255ad3-c009-4055-901f-c20f1d269cd5)
![Screenshot 2023-05-20 100643](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/baadf8ab-93ee-4ecb-b550-45a9b542ed73)
![Screenshot 2023-05-20 100652](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/6fcff7b2-ee21-4825-813a-07ea24083d81)

**Comment by JPH71** [comment]
Actors name with ' sorted...
I will work on the series name... though I have already sorted carnal+
studio appearing as an actor...

Thanks for finding this error...

Jason xxx

**Comment by JPH71** [comment]
I have also sorted out the series thing too... that's if it has more than 3
words...

**Comment by JPH71** [comment, marked as answer]
Just testing... hopefully will be with Cody tomorrow  xxx

**Comment by dimmitri9** [comment]
Don't know why this is happening, but the updated plugins aren't matching any titles from now on

[com.plexapp.agents.WayBig.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11560637/com.plexapp.agents.WayBig.log)
[com.plexapp.agents.Fagalicious.log](https://github.com/CodyBerenson/PGMA-Modernized/files/11560641/com.plexapp.agents.Fagalicious.log)

https://fagalicious.com/trystanbull-trystan-bull-nick-reeves-and-max-morgan-in-my-stepdad-banged-me/
https://www.waybig.com/blog/2022/11/16/gayhoopla-gio-givannis-first-time-trying-ass-play-on-camera/
https://fagalicious.com/idisruptive-films-ty-roderick-michael-boston-first-timers/
